tools/webpack/packages.js

```diff
@@ -152,6 +152,16 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 		to: join( baseDir, `src/${ phpFiles[ filename ] }` ),
 	} ) );
 
+	const phpMetadataCopies = Object.keys( phpFiles ).map( function( filename ) {
```

Comment:
It's a block metadata.
```suggestion
	const blockMetadataCopies = Object.keys( phpFiles ).map( function( filename ) {
```